llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -21378,6 +21382,9 @@ SDValue RISCVTargetLowering::joinRegisterPartsIntoValue(
     return Val;
   }
 
+  if (ValueVT.isRISCVVectorTuple())
```

Comment:
Why do we need this now?

Reply:
Actually we can remove it after removing tuple from scalable vector type.